build-system/compile/compile.js

```diff
@@ -186,7 +186,11 @@ function compile(
         // custom-elements polyfill must be included.
         // install intersection-observer to esm build as iOS safari 11.1 to
         // 12.1 do not have InObs.
```

Comment:
Can we clean up this comment?

Reply:
I'm removing this in a follow up PR, just trying to get the cherry-pick fix out as quick as possible.